.travis.yml

```diff
@@ -60,14 +12,90 @@ before_install:
   - chmod +x docker-compose
   - sudo mv docker-compose /usr/local/bin
 
-before_script:
-  - if [ "$CC" == "gcc" ]; then export CC=gcc-4.8; fi
-  - if [ "$CXX" == "g++" ]; then export CXX=g++-4.8; fi
-  - if [ "$CC" == "clang" ]; then export CC=clang-3.8; fi
-  - if [ "$CXX" == "clang++" ]; then export CXX=clang++-3.8; fi
-  - docker-compose build base
-  - if [ "$TARGET" == "android" ]; then docker-compose build java; fi
-  - docker-compose build ${TARGET}
-
-script:
-  - docker-compose run ${TARGET}
+jobs:
+  fast_finish: true
+  allow_failures:
+    - env: TARGET=sonar-scan
```

Comment:
Sonar env vars are stored in our Travis CI settings page. Those settings are not shared with external PRs, for security reasons. The sonar-scan.sh script check if those variables exists and fails if they don't. So for a PR, or for a fork without the correct sonar scan configuration, this will fail. 

So, this is an allowed failure to prevent having failed builds for every external PR and fork